### PR TITLE
Issue 11 - getActionsByAddress

### PR DESCRIPTION
**Pull request by Fabryprog**
#11

### CODE REVIEW

File: src/test/java/org/iotexproject/antenna/rpcmethod/IoTeXGRPCTestInterface.java [hunk 1]
```diff
@@ -1,5 +1,15 @@
 package org.iotexproject.antenna.rpcmethod;
 
+/**
```
Comment (ququzone) [inline src/test/java/org/iotexproject/antenna/rpcmethod/IoTeXGRPCTestInterface.java]: Can Java LICENSE comment is this style?

**Comment by ququzone** [comment]
Please rebase code without conflicts.

File: src/test/java/org/iotexproject/antenna/rpcmethod/IoTeXGRPCTestInterface.java (hunk 1 above)
Comment (Fabryprog) [reply src/test/java/org/iotexproject/antenna/rpcmethod/IoTeXGRPCTestInterface.java]: What do you mean? This is like same java source file!

**Comment by Fabryprog** [comment]
> Please rebase code without conflicts.

the code hasn't any conflics! it is ok!

**Comment by ququzone** [comment]
> > Please rebase code without conflicts.
> 
> the code hasn't any conflics! it is ok!

Screen Shot 2019-04-29 at 7 48 02 AM

**Comment by Fabryprog** [comment]
> > > Please rebase code without conflicts.
> > 
> > 
> > the code hasn't any conflics! it is ok!
> 
> Screen Shot 2019-04-29 at 7 48 02 AM

This is very strange

image

**Pull request merged**